.travis.yml

```diff
@@ -35,7 +35,7 @@ after_success:
 before_deploy:
 - INSTALL4J_HOME=~/install4j
 - ./.travis/install_install4j "$INSTALL4J_HOME"
-- ./gradlew -Pinstall4jHomeDir="$INSTALL4J_HOME" release
+- ./gradlew -Pinstall4jHomeDir="$INSTALL4J_HOME" release | egrep -v "^WARNING: Passing class file uncompressed due to unrecognized attribute|Pack200Logger warning$"
```

Comment:
Hmm thinking about this I'm concerned this may cause silent errors, in case the gradle command fails, the egrep command (I'm recall a discussion between `egrep` and `grep -E`), will still be executed, and thus the deployment only fails when the `egrep` command fails as well

Reply:
That's a really good consideration of exit codes. We should get a failure downstream if the installer files are missing, but I would agree that is not a good failure mode to rely on.

I tested out some variants on came up with this:
```
 (./gradlew -Pinstall4jHomeDir="$INSTALL4J_HOME" release  2> errors && echo "INSTALL4J finished with $(wc -l errors)") || (echo "INSTALL4J FAILED" && tail -500 errors) 
```

It turns out the `egrep` was not going to do what we wanted anyways as the warnings were going to stderr.

With the above, on the success case we see:


```
:
:
Compressed media file 'Mac OS X':
  Compressing files
  Applying LZMA compression
  Generating VM options file vmoptions.txt.
  Signing installer (adhoc)
  Moving media files to media directory /home/dan/work/triplea/game-headed/build/releases
  The name of the media file is TripleA_1.10.dev_macos.dmg.
  The size of the media file is 63.2 MB
Writing updates.xml.
Build completed in 81.8 seconds.

BUILD SUCCESSFUL in 2m 52s
38 actionable tasks: 38 executed
INSTALL4J finished with 13209 errors
```

Testing out a case where install4j fails, we get the following:
```
:
:
WARNING: Passing class file uncompressed due to unrecognized attribute: javafx/beans/property/ReadOnlyMapWrapper.class
Aug 23, 2019 11:38:42 AM com.sun.java.util.jar.pack.Utils$Pack200Logger warning
WARNING: Passing class file uncompressed due to unrecognized attribute: javafx/beans/property/ReadOnlyListProperty.class
Aug 23, 2019 11:38:42 AM com.sun.java.util.jar.pack.Utils$Pack200Logger warning
WARNING: Passing class file uncompressed due to unrecognized attribute: javafx/beans/property/ReadOnlyStringProperty.class
Aug 23, 2019 11:38:42 AM com.sun.java.util.jar.pack.Utils$Pack200Logger warning
WARNING: Passing class file uncompressed due to unrecognized attribute: javafx/beans/property/MapPropertyBase.class
Aug 23, 2019 11:38:42 AM com.sun.java.util.jar.pack.Utils$Pack200Logger warning
WARNING: Passing class file uncompressed due to unrecognized attribute: javafx/beans/property/ReadOnlyMapPropertyBase.class
Aug 23, 2019 11:38:42 AM com.sun.java.util.jar.pack.Utils$Pack200Logger warning
WARNING: Passing class file uncompressed due to unrecognized attribute: javafx/beans/property/ReadOnlyObjectWrapper$1.class
Aug 23, 2019 11:38:42 AM com.sun.java.util.jar.pack.Utils$Pack200Logger warning
WARNING: Passing class file uncompressed due to unrecognized attribute: javafx/beans/property/StringProperty.class
Aug 23, 2019 11:38:42 AM com.sun.java.util.jar.pack.Utils$Pack200Logger warning
WARNING: Passing class file uncompressed due to unrecognized attribute: javafx/beans/NamedArg.class
Aug 23, 2019 11:38:42 AM com.sun.java.util.jar.pack.Utils$Pack200Logger warning
WARNING: Passing class file uncompressed due to unrecognized attribute: javafx/beans/DefaultProperty.class
javax.imageio.IIOException: Can't read input file!
	at java.desktop/javax.imageio.ImageIO.read(ImageIO.java:1308)
	at com.install4j.b.c.b(ejt:376)
	at com.install4j.b.c.b(ejt:336)
	at com.install4j.b.b.a(ejt:136)
	at com.install4j.b.i.a(ejt:414)
	at com.install4j.b.i.b(ejt:156)
	at com.install4j.Install4JApplication.c(ejt:479)
	at com.exe4j.a.a(ejt:338)
	at com.install4j.Install4JApplication.main(ejt:94)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at com.exe4j.runtime.LauncherEngine.launch(LauncherEngine.java:85)
	at com.install4j.runtime.launcher.UnixLauncher.main(UnixLauncher.java:62)
install4j: compilation failed. Reason: java.io.IOException: Image file "/home/dan/work/triplea/game-headed/build/assets/icons/triplea_icon_16_16.png" for application icon does not exist.

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':game-headed:platformInstallers'.
> Process 'command '/bin/sh'' finished with non-zero exit value 1
```

The tail -500 should help guarantee we can see the last lines that killed the install4j process and we'd avoid clogging up the travis log (which importantly avoids log truncation).